R/feature_spec.R

```diff
@@ -949,6 +949,36 @@ StepTextEmbeddingColumn <- R6::R6Class(
 )
 
 
+# StepImageEmbedding -----------------------------------------------------
+
+StepImageEmbeddingColumn <- R6::R6Class(
+  "StepTextEmbeddingColumn",
```

Comment:
this looks like left over from the other one :-)

Reply:
thanks!